services/azure-devops/azure-devops-coverage.tester.js

```diff
@@ -0,0 +1,215 @@
+'use strict'
+
+const Joi = require('joi')
+const { withRegex } = require('../test-validators')
+const t = require('../create-service-tester')()
+module.exports = t
+
+const service = 'azure-devops'
+const noun = 'coverage'
+const org = 'swellaby'
+const project = 'opensource'
+const linuxDefinitionId = 21
+const macDefinitionId = 26
+const windowsDefinitionId = 24
+const nonExistentDefinitionId = 234421
+const buildId = 946
+const uriPrefix = `/${service}/${noun}/${org}/${project}`
+const azureDevOpsApiBaseUri = `https://dev.azure.com/${org}/${project}/_apis`
+const mockBadgeUriPath = `${uriPrefix}/${macDefinitionId}.json`
+const mockLatestBuildApiUriPath = `/build/builds?definitions=${macDefinitionId}&$top=1&api-version=5.0-preview.4`
+const mockCodeCoverageApiUriPath = `/test/codecoverage?buildId=${buildId}&api-version=5.0-preview.1`
+const percentageRegex = /^[1-9][0-9]?%|^100%|^0%$/
```

Comment:
This looks like a better version of `isIntegerPercentage` from `test-validators`. 😁Could you use + update that regex instead?

Reply:
Sigh.. I checked test-validators at least 4 times last night and somehow managed to miss `isIntegerPercentage`  🤦‍♂️ Will make the updates